vmm/src/default_syscalls/x86_64.rs

```diff
@@ -23,6 +23,7 @@ pub const ALLOWED_SYSCALLS: &[i64] = &[
     libc::SYS_futex,
     libc::SYS_ioctl,
     libc::SYS_lseek,
+    libc::SYS_madvise,
```

Comment:
As we discussed offline, it looks like only musl requires this syscall. Since there is already an effort to add seccomp filters for glibc as well (#888) I would suggest using cfg(musl) for this one.